.github/workflows/full_suite_integration_tests.yml

```diff
@@ -99,12 +99,14 @@ jobs:
 
       - name: Prepare Google Service Account
         env:
+          FLANK_PROJECT_ID: flank-open-source
+          GOOGLE_CLOUD_PROJECT: flank-open-source
           GCLOUD_KEY: ${{ secrets.GCLOUD_KEY }}
         run: |
             if [ "$RUNNER_OS" == "Windows" ]; then
                 set GCLOUD_DIR="%HOMEPATH%/.config/gcloud/"
                 mkdir %GCLOUD_DIR%
-                echo certutil -decode gcloud_key.txt %GCLOUD_DIR%application_default_credentials.json
+                echo certutil -decode %GCLOUD_KEY% %GCLOUD_DIR%application_default_credentials.json
```

Comment:
if this work, let's remove
```
FLANK_PROJECT_ID: flank-open-source
GOOGLE_CLOUD_PROJECT: flank-open-source
```

Reply:
We don't need these additional variables because ```application_default_credentials.json``` contains this data

Reply:
removed.